flink-core/src/main/java/org/apache/flink/api/common/state/StateTtlConfiguration.java

```diff
@@ -93,4 +101,59 @@ public Time getTtl() {
 	public TtlTimeCharacteristic getTimeCharacteristic() {
 		return timeCharacteristic;
 	}
+
+	@Override
+	public String toString() {
+		return "StateTtlConfiguration{" +
+			"ttlUpdateType=" + ttlUpdateType +
+			", stateVisibility=" + stateVisibility +
+			", timeCharacteristic=" + timeCharacteristic +
+			", ttl=" + ttl +
+			'}';
+	}
+
+	public static Builder newBuilder() {
+		return new Builder();
+	}
+
+	/**
+	 * Builder for the {@link StateTtlConfiguration}.
+	 */
+	public static class Builder {
+
+		private TtlUpdateType ttlUpdateType;
```

Comment:
I would initialise these fields with some reasonable default values, like `OnCreateAndWrite`, `NeverReturnExpired`, and `ProcessingTime`.